model/qti/AssetParser.php

```diff
@@ -356,6 +360,16 @@ private function getXmlProperties($properties)
         return $xmls;
     }
 
+    private function extractAdvancedCustomInteractionAssets(CustomInteraction $interaction): void
+    {
+        $extractorAllocator = $this->getCustomInteractionAssetExtractorAllocator();
+        $extractor = $extractorAllocator->allocateExtractor($interaction->getTypeIdentifier());
+
+        foreach ($extractor->extract($interaction) as $asset) {
+            $this->addAsset('apip', $asset);
```

Comment:
Why all these extractors would add assets as `apip`? `apip` is a specific type of accessibility related stuff http://www.imsglobal.org/apip/index.html

Reply:
i was trying to define type of asset by mime-type from data url, but it was increased complexity of the component by supporting mapping mime types to assets types. and i decide to use something common, cause we just need to push in assets result array and we have restriction of supported assets type. Do you see any critical problems in this case? maybe you can advise something for that usage?

Reply:
Well, you could use a [map from elsewhere](https://github.com/thephpleague/mime-type-detection/blob/main/src/GeneratedExtensionToMimeTypeMap.php), but I agree there is no great benefit to learn the exact asset type here. Yet `apip` is a bit misleading. You used term `advanced` in the method name to differentiate theses extracted assets, maybe it could be used as a type here as well

Reply:
usage of some new type there create a need to extend this map https://github.com/oat-sa/extension-tao-item/blob/master/models/classes/pack/ItemPack.php#L48

Reply:
Okay, but it doesn't change the fact that `apip` type here is incorrect and it's a shortcut. If no time to refactor, by returning a proper type or creating a new type, I think it would be better to add a comment why `apip` type was used here